### ISSUE TITLE
Add option to apply material asset in edit mode

### ISSUE THREAD

**Issue by c-d-a** [Gorgious56/asset_browser_utilities#103]
For some reason, Blender currently lacks the ability to assign an asset to selected faces.  
Based in large part on your stackexchange answers, I've made such an operator for my own use https://github.com/c-d-a/io_import_wad2/commit/fdb46588a7acd4eecccc0e255fda7f596d8e91bf
While your addon is more focused on batch operations, seems like it would fit.

**Comment by Gorgious56** [comment]
Ooh nice, yeah there are still some utilities that need to be added to vanilla blender but until then... thanks for sharing ;) I gather you don't mind me adding it to my addon then ?

Cheers

**Comment by c-d-a** [comment]
Yeah, feel free to lift wholesale, I've basically lifted it from you to begin with.
